tests/Fixtures/Entity/SluggableWithUniqenessAndOwnRepositoryEntity.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Knp\DoctrineBehaviors\Tests\Fixtures\Entity;
+
+use Doctrine\ORM\Mapping as ORM;
+use Knp\DoctrineBehaviors\Contract\Entity\SluggableInterface;
+use Knp\DoctrineBehaviors\Model\Sluggable\SluggableTrait;
+
+/**
+ * @ORM\Entity(repositoryClass="Knp\DoctrineBehaviors\Tests\Fixtures\Repository\SluggableWithUniqenessAndOwnRepositoryRepository")
```

Comment:
This should work without dependency on specific repository.

```suggestion
 * @ORM\Entity()
```